src/server/pegasus_write_service.cpp

```diff
@@ -36,111 +124,36 @@ pegasus_write_service::pegasus_write_service(pegasus_server_impl *server)
       _server(server),
       _impl(new impl(server)),
       _batch_start_time(0),
-      _cu_calculator(server->_cu_calculator.get())
+      _cu_calculator(server->_cu_calculator.get()),
```

Comment:
How about use macros to format/simplify these codes? (Maybe the member variables and metrics should have some similar naming?)

Reply:
OK, I'll add a new macro system to simplify/standardize operations for metric variables.